.github/workflows/build-packages.yml

```diff
@@ -546,3 +546,131 @@ jobs:
+        wc jans-sbom-fsct-report.json 
+
+        # generate BSI TR-03183-2 (v2.0.0) compliance report
+        sbomqs compliance --bsi-v2 -j jans-enriched-sbom.json > jans-sbom-bsi-v2-report.json
+        # To confirm the file is generated
+        wc jans-sbom-bsi-TR0318-v2-report.json
+
+        # generate NTIA minimum elements (July 12, 2021) compliance report
+        sbomqs compliance --ntia -j jans-enriched-sbom.json > jans-sbom-NTIA-report.json
+        # To confirm the file is generated
+        wc jans-sbom-NTIA-report.json
+
+        # generate OpenChain Telco SBOM (v1.0) compliance report
+        sbomqs compliance --oct -j jans-enriched-sbom.json > jans-sbom-openchain-report.json
+        # To confirm the file is generated
+        wc jans-sbom-openchain-report.json
+
+
+    # Upload the enriched SBOM and compliance reports to the release
+    - name: Upload the enriched SBOM 
+      uses: svenstaro/upload-release-action@1beeb572c19a9242f4361f4cee78f8e0d9aec5df # v2
+      with:
+        repo_token: ${{ secrets.MOAUTO_WORKFLOW_TOKEN }}
+        file: jans-enriched-sbom.json
+        asset_name: Software Bill Of Materials(SBOM)
+        tag: ${{ steps.previoustag.outputs.tag }}
+        overwrite: true
+
+    # Upload the fsct report to the release
+    - name: Upload the fsct report to the release
+      uses: svenstaro/upload-release-action@1beeb572c19a9242f4361f4cee78f8e0d9aec5df # v2
+      with:
+        repo_token: ${{ secrets.MOAUTO_WORKFLOW_TOKEN }}
+        file: jans-sbom-fsct-report.json
+        asset_name: FSCT compliance report
+        tag: ${{ steps.previoustag.outputs.tag }}
+        overwrite: true
+
+    # Upload the BSI TR-03183 v2 report to the release
+    - name: Upload the BSI TR-03183 v2 to the release
+      uses: svenstaro/upload-release-action@1beeb572c19a9242f4361f4cee78f8e0d9aec5df # v2
+      with:
+        repo_token: ${{ secrets.MOAUTO_WORKFLOW_TOKEN }}
+        file: jans-sbom-bsi-TR0318-v2-report.json
+        asset_name: BSI TR-03183 v2 compliance report
+        tag: ${{ steps.previoustag.outputs.tag }}
+        overwrite: true
+
+    # Upload the NTIA report to the release
+    - name: Upload the NTIA to the release
+      uses: svenstaro/upload-release-action@1beeb572c19a9242f4361f4cee78f8e0d9aec5df # v2
+      with:
+        repo_token: ${{ secrets.MOAUTO_WORKFLOW_TOKEN }}
+        file: jans-sbom-NTIA-report.json
+        asset_name: NTIA compliance report
+        tag: ${{ steps.previoustag.outputs.tag }}
+        overwrite: true
+
+    # Upload the OpenChain Telco SBOM report to the release
+    - name: Upload the OpenChain Telco SBOM to the release
+      uses: svenstaro/upload-release-action@1beeb572c19a9242f4361f4cee78f8e0d9aec5df # v2
+      with:
+        repo_token: ${{ secrets.MOAUTO_WORKFLOW_TOKEN }}
+        file: jans-sbom-openchain-report.json
+        asset_name: OpenChain Telco SBOM compliance report
+        tag: ${{ steps.previoustag.outputs.tag }}
+        overwrite: true
+
```

Comment:
Remove and use gh upload instead. This can be a one liner instead of a multiple step